content/2016.06.ab-testing-with-kinto.rst

```diff
@@ -0,0 +1,199 @@
+How to use Kinto to A/B test a web application?
+###############################################
+
+:lang: en
+:date: 2016-06-05
+:url: ab-testing-with-kinto
+:summary: A/B testing is a form of statistical hypothesis testing. Its
+          goal is to identify changes to web pages that increase or
+          maximize an outcome of interest. Let's see how we can use
+          kinto to use this technic on our website.
+
+
+Introduction
+============
+
+I am not a big fan of A/B testing because I think, most of the time,
+`we can do better`_ however in `certain circumstances`_ A/B testing is
+still a really good idea to test a feature.
+
+.. image:: {filename}/images/kinto-logo.png
+    :alt: The Kinto Logo
+
+
+In an other hand, I am a big fan of the philosophy behind Kinto_ which
+as you may know is a database for the web.
```

Comment:
Oh that's the new way of defining Kinto? Not a service that can store JSON data?